guide/how-it-works/sign-in-with-bitcoin.md

```diff
@@ -0,0 +1,161 @@
+
+### Wallet-service communication
+
+Your service and the user wallet need to be able to communicate with each other to exchange information. There are two basic ways to handle this—direct and bridged.
+
+#### Direct communication
+
+The smoothest user experience is when the wallet and the service can directly communicate with each other. For example, a browser wallet and a website can send messages back and forth (see [WebLN](https://www.webln.guide/introduction/readme)). This allows for convenient features like allowances for automatic approval of payments for services the user trusts.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/direct-connection.png"
+   retina = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/direct-connection@2x.png"
+   alt-text = "Browser window with a wallet extension installed"
+   width = 800
+   height = 411
+%}
+
+#### Bridged communication
+
+There is more friction if the user needs to be involved in this communication. However, this extra friction provides more user control and better security. In the visualization below, the service and user wallet cannot directly communicate with each other. Instead, the web application presents instructions as QR codes that the user needs to scan with their (mobile) wallet. The wallet then sends instructions to the application server, which updates the web application.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/bridged-connection.png"
+   retina = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/bridged-connection@2x.png"
+   alt-text = "Communication flow between a browser window and a smart phone wallet"
+   width = 800
+   height = 411
+%}
+
+This does not necessarily have to be done via QR code. The most basic method could even involve manually copying and pasting messages and identifiers back and forth.
+
+## Signing in
+
+The biggest hurdle at this stage is that most users will not be familiar with this authentication method and are not likely to have a wallet they can readily use. Ensure that there are concise explanations and tips to get users started.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/signin-info.png"
+   retina = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/signin-info@2x.png"
+   alt-text = "Sign in screen with tips for wallets to use and general information"
+   width = 800
```

Comment:
Are these two images representing the different types of communication (bridged and direct)? If so, stating which is which along side both image would be handy.

Reply:
Added caption to clarify that one is the sign-in prompt from the website and the other is the user confirming the sign-in in the wallet.